FirestoreLINQ/CustomAttributes/FireStoreCollectionAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
+using FirestoreLINQ.Internals;
+
+// ReSharper disable once CheckNamespace
+namespace FirestoreLINQ
+{
+    public class FireStoreCollectionAttribute : Attribute
```

Comment:
`[AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]`
Add this attribute on top of class name.
This will only allow users to add the attribute on the Class name and also only once on a single class.